src/hooks/useElectronicKey.js

```diff
@@ -159,7 +161,11 @@ export default (function useElectronicKey() {
                 cleanup()
             }
             currentPromise = currentPromise.then(() => {
-                return playWithSpaces(localQueue[i])
+                var ditDah = localQueue[i]
+                if (ditDah)
```

Comment:
BTW, this seemed to be a bug, causing `undefined` to be passed to `playWithSpaces(...)` which was causing an extra pause.